src/lib.rs

```diff
@@ -22,15 +20,38 @@
 //!     }
 //! }
 //! ```
-#[macro_use] extern crate error_chain;
-#[macro_use] extern crate log;
-extern crate errno;
-extern crate kmod_sys;
+#![deny(
```

Comment:
I'd prefer to drop this one since this technically breaks rusts stability. Usually the code is guaranteed to compile with all future rust 1.0 compilers, but code that's valid now may cause warnings in a future rust compiler. Making this a hard failure could break this crate an all crates depending on it.